client/my-sites/site-settings/date-time-format/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import HeaderCake from 'components/header-cake';
+import { getSelectedSite } from 'state/ui/selectors';
+
+export const DateTimeFormat = ( { translate, site } ) => {
+	const goBack = () => {
+		page( '/settings/general/' + site.slug );
+	};
+
+	return (
+		<div className="main main-column date-time-format" role="main">
+			<DocumentHead title={ translate( 'Manage Date and Time Format Configurations' ) } />
```

Comment:
@yoavf I guess you're here for the wrong comment, but please don't waste your time on this PR already, since it's still very WIP.
I'll likely move things around a bit in the next commits, and I'm sure when I'll get some design feedback the translation strings will change too.

Maybe it could make sense to limit the i18n bot to ready to merge PRs only?
(I'd say to "needs i18n review" only, but I know us devs, we always forget about that label 🙂 )